### PR TITLE
use submodule that has roslint fixes except for the line length, references, and function size

### PR DESCRIPTION
**Pull request by lucasw**
Paired with subrepo PR https://github.com/LORD-MicroStrain/microstrain_inertial_driver_common/pull/16

### CODE REVIEW

File: microstrain_inertial_driver/CMakeLists.txt
Comment (robbiefish) [inline microstrain_inertial_driver/CMakeLists.txt:185]: This looks good, but I think a good thing to do here to hopefully catch these errors before they get merged would be to run roslint on the buildfarm, which could be done by adding the following code snippet around this line:
```cmake
if(BUILD_TESTING)
  set_property(TARGET roslint PROPERTY EXCLUDE_FROM_ALL OFF)
endif()
```

And then somewhere near the top of the `CMakeLists.txt`, add the line
```cmake
option(BUILD_TESTING "Whether or not to build tests" OFF)
```

File: .gitmodules
Comment (robbiefish) [inline .gitmodules]: Now that the submodule PRs have been merged, the changes to this file should be reverted

**Review by robbiefish — CHANGES_REQUESTED** [review]

File: microstrain_inertial_driver/CMakeLists.txt
Comment (robbiefish) [reply microstrain_inertial_driver/CMakeLists.txt:185]: I still think this would be a good thing to do, but I merged some code changes that broke roslint before this, so this can be implemented in a second PR so that this does not cause this PR to fail

**Comment by robbiefish** [comment]
@lucasw I am going to go ahead and merge this, if there are more changes you would like to add, please feel free to open another PR

**Review by robbiefish — APPROVED** [review]

File: microstrain_inertial_driver/CMakeLists.txt
Comment (lucasw) [reply microstrain_inertial_driver/CMakeLists.txt:185]: Ok following up with https://github.com/LORD-MicroStrain/microstrain_inertial/pull/154

**Pull request merged**
